build.sbt

```diff
@@ -34,27 +33,20 @@ lazy val squants =
   .in(file("."))
   .settings(defaultSettings: _*)
   .jvmConfigure(
-    _.enablePlugins(TutPlugin, SbtOsgi)
+    _.enablePlugins(MdocPlugin, SbtOsgi)
```

Comment:
By enabling the `MdocPlugin` in the library, this adds the `mdoc` artifacts as dependencies of squants itself. I don’t think this should be the case.

![Screenshot from 2021-05-06 16-37-12](https://user-images.githubusercontent.com/332812/117316966-600ab880-ae89-11eb-8add-6c4b84c26ac5.png)

Instead, the `MdocPlugin` should be enabled to a separate “documentation” project.

Reply:
I’ve tried to address this problem but I faced other issues. The most important one is that the current README.md does not compile with mdoc.

Reply:
I’ve created #467 to fix this problem.